docker-compose-deploy.yaml

```diff
@@ -1,20 +1,17 @@
 version: '2.2'
 
 x-jellyfish-template: &jellyfish-template
-  image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-0.1.0-4a3173b}"
+  image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-0.1.0-5ef4452}"
   restart: unless-stopped
   environment: &jellyfish-environment
-    JF_INTEGRATED_TURN_LISTEN_IP: "0.0.0.0"
-    JF_INTEGRATED_TURN_PORT_RANGE: "50000-50500"
-    JF_INTEGRATED_TCP_TURN_PORT: "49999"
-    JF_SERVER_API_TOKEN: ${SERVER_API_TOKEN}
-    RELEASE_DISTRIBUTION: name
-    RELEASE_COOKIE: ${SERVER_API_TOKEN}
     JF_PORT: 5002
-    JF_NODES: "app@jellyfish1 app@jellyfish2"
-    JF_CHECK_ORIGIN: "${CHECK_ORIGIN:-true}"
-    BEAM_PORT: 9000
-    ELIXIR_ERL_OPTIONS: "-kernel inet_dist_listen_min $BEAM_PORT inet_dist_listen_max $BEAM_PORT"
```

Comment:
Where do we limit ports for distribution?

Reply:
In Jellyfish Dockerfile https://github.com/jellyfish-dev/jellyfish/blob/main/Dockerfile#L113-L114